docs/labwc-actions.5.scd

```diff
@@ -117,16 +117,26 @@ Actions are used in menus and keyboard/mouse bindings.
 	upper-left corner of the window associated with the action. Default is
 	yes.
 
-*<action name="ToggleDecorations" />*
+*<action name="ToggleDecorations" keepBorder="yes" />*
```

Comment:
Placeholder text uses `value` or an empty string rather than one of the specific options.

```suggestion
*<action name="ToggleDecorations" keepBorder="value" />*
```